src/fisher.jl

```diff
@@ -71,7 +71,8 @@ immutable FisherExactTest <: HypothesisTest
 end
 
 testname(::FisherExactTest) = "Fisher's exact test"
-population_param_of_interest(x::FisherExactTest) = ("Odds ratio", 1.0, x.ω) # parameter of interest: name, value under h0, point estimate
+population_param_of_interest(x::FisherExactTest) = ("Odds ratio", 1.0, x.ω)
```

Comment:
Better not change this since that's unrelated. Or at least put the comment before the function rather than after. Same below.